analysers/trials/databaseSize.py

```diff
@@ -4,32 +4,49 @@
 import gzip
 import uuid
 
+from datetime import timedelta
+
 from pyspark_cassandra import CassandraSparkContext
 from pyspark import SparkConf
 
-# Takes arguments: Spark master, Cassandra host, Minio host, path of the files
-sparkMaster = sys.argv[1]
-cassandraHost = sys.argv[2]
-trialID = sys.argv[3]
-experimentID = trialID.split("_")[0]
-cassandraKeyspace = "benchflow"
-srcTable = "database_sizes"
-destTable = "trial_byte_size"
-
-# Set configuration for spark context
-conf = SparkConf() \
-    .setAppName("Database size analyser") \
-    .setMaster(sparkMaster) \
-    .set("spark.cassandra.connection.host", cassandraHost)
-sc = CassandraSparkContext(conf=conf)
-
-data = sc.cassandraTable(cassandraKeyspace, srcTable)\
-        .select("size") \
-        .where("trial_id=? AND experiment_id=?", trialID, experimentID) \
-        .reduce(lambda a, b: a["size"]+b["size"])
+def databaseSize(dataRDD):
+    if dataRDD.isEmpty():
+        return None
+    data = dataRDD.reduce(lambda a, b: a["size"]+b["size"])
+    if type(data) is int:
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Using type() instead of isinstance() for a typecheck.](https://www.codacy.com/app/BenchFlow/analysers/file/2612093444/issues/source?bid=3301029&fileBranchId=3301030#l16)